src/org/rascalmpl/parser/gtd/result/SkippedNode.java

```diff
@@ -8,50 +8,65 @@
  * Contributors:
 
  *   * Arnold Lankamp - Arnold.Lankamp@cwi.nl
+ *   * Pieter Olivier - Pieter.Olivier@swat.engineering
```

Comment:
Thanks, After I made this change I found out that newer files have a different license header without the `Contributors` section. After that I did not really see a point in adding my name in older files that I happend to contribute to.

Reply:
In that case, consider adding your name nowhere. It's really a minor point that most people won't care about, I reckon, but if your name is present in some files, but absent in others, implicitly, it might give the impression that you didn't contribute to the files where your name is *not-present*; i.e., absence in one place could be construed as semantically meaningful in combination with presence in another place. In contrast, if your name is absent everywhere, in my mind, it remains unclear if you didn't contribute to any file or simply didn't update the contributor lists.

But again, I suppose I'm overthinking... 😉